messaging/tests/snapshots/snap_test_api.py

```diff
@@ -76,17 +76,17 @@
                     'edges': [
                         {
                             'node': {
-                                'time': '1970-12-29T14:27:50.629900+00:00'
+                                'time': '1970-12-29T14:07:50.629900+00:00'
```

Comment:
Why does the snapshot now have 20 minutes difference to old times?

Reply:
Details in commit fc8e61820345a0ebeffe927de8560638725f6e32 message

Reply:
See also https://github.com/City-of-Helsinki/kukkuu/pull/480#issuecomment-2671080976